src/scriptsys/lib_game.c

```diff
@@ -0,0 +1,131 @@
+#include "lua/lua.h"
+#include "lua/lualib.h"
+#include "lua/lauxlib.h"
+#include "scriptsys/script.h"
+#include "common/console.h"
+#include "containers/hash_set.h"
+#include "allocator/allocator.h"
+#include "lib_log.h"
+#include "lib_game.h"
+
+static HashSet update_handlers;
```

Comment:
Would like to have ms_ prefix so that its clear this is a static variable rather than a local variable.